notebooks/00_core.ipynb

```diff
@@ -30,6 +30,7 @@
     "import os\n",
```

Comment:
<ul><li>add <code>plt.show()</code>  to avoid the ugly matplotlib output</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>